stdlib/public/SDK/Foundation/NSStringAPI.swift

```diff
@@ -156,7 +170,7 @@ extension String {
   /// C array of UTF8-encoded bytes.
   public init?(utf8String bytes: UnsafePointer<CChar>) {
     if let ns = NSString(utf8String: bytes) {
-      self = ns as String
+      self = String._unconditionallyBridgeFromObjectiveC(ns)
```

Comment:
We need to continue the conversation about Linux bridging, but separately from this commit.